NetKAN/StationKeeping.netkan

```diff
@@ -0,0 +1,8 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "StationKeeping",
+    "$kref"        : "#/ckan/github/swilbur/StationKeeping",
+    "license"      : "CC-BY-4.0",
```

Comment:
Please add an abstract describing the mod, such as:

    "abstract": "Adds a button in the Tracking Station to set a ship's orbit. If you are with 1% (configurable) of your target orbit, you can simply push the button to set the orbit's semi-major axis to exactly what you want, so it won't drift out of alignment as you time warp.",